aiida/cmdline/commands/cmd_export.py

```diff
@@ -157,11 +158,11 @@ def migrate(input_file, output_file, force, silent, archive_format):
 
         new_version = verify_metadata_version(metadata)
 
-        with open(folder.get_abs_path('data.json'), 'w') as handle:
-            json.dump(data, handle)
+        with io.open(folder.get_abs_path('data.json'), 'wb', encoding=None) as fhandle:
```

Comment:
dito

Reply:
there are even more occurrences of the same, but if you want to leave them as they are, that's fine for me as well

Reply:
As above, aiming for the 'explicit is better than implicit' approach.